cogs/committee_actions_tracking.py

```diff
@@ -281,7 +288,7 @@ async def create(
         required=True,
         parameter_name="status",
     )
-    @CommandChecks.check_interaction_user_has_committee_role
+    @CommandChecks.check_interaction_user_has_committee_or_elect_role
```

Comment:
rather than going through the effort of creating this new check, we can simply remove the committee check - since non-committee and non-cmt-elect can still hold actions and they should be able to change the status of their own action